apps/server/src/modules/board/service/content-element-update.visitor.ts

```diff
@@ -59,7 +59,8 @@ export class ContentElementUpdateVisitor implements BoardCompositeVisitor {
 
 	visitSubmissionContainerElement(submissionContainerElement: SubmissionContainerElement): void {
 		if (this.content instanceof SubmissionContainerContentBody) {
-			submissionContainerElement.dueDate = this.content.dueDate ?? undefined;
+			if (this.content.dueDate === undefined) return;
```

Comment:
I am not a fan of ifs without { and returns on the same lines, because it makes the code harder to read.
if (false === this.content instanceof SubmissionContainerContentBody) {
  this.throwNotHandled(submissionContainerElement);
}
if (this.content.dueDate) {
  submissionContainerElement.dueDate = this.content.dueDate;
}

Reply:
Interesting, I'm the opposite. I want early exits.

Also we don't want to throw. We just don't want to skip the node when dueDate = undefined.
We only want to update when dueDate is set explicitly to Date or null.